addons/l10n_it_edi_sdicoop/tests/test_edi_reverse_charge_xml.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import datetime
+import logging
+from lxml import etree
+
+from odoo.tests import tagged
+from odoo.addons.l10n_it_edi_sdicoop.tests.test_edi_xml import TestItEdi
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestItEdiReverseCharge(TestItEdi):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        def get_tags(*tag_codes):
+            return cls.env['account.account.tag'].search([('applicability', '=', 'taxes'), ('name', 'in', tag_codes)]).ids
+
+        def rep_lines(*lines):
+            return [(5, 0, 0)] + [
+                (0, 0, {
+                    'factor_percent': line[0],
+                    'repartition_type': line[1],
+                    'tag_ids': get_tags(*line[2:]) if len(line) > 2 else False
+                })
+                for line in lines
+            ]
+
+        def prod_line(*lines):
+            return [(0, 0, {**line[0], 'name': line[1], 'product_id': line[2]}) for line in lines]
```

Comment:
Couldn't we use namedtuple or dict?
It applies for rep_lines function too.

Reply:
![immagine](https://user-images.githubusercontent.com/1665365/176423465-041daf67-1466-4215-8ee0-119f85e2bd60.png)
If I was to repeat dict keys all over, the file would be huge and unreadable imho.
Imho, the part above the red sign is clearer, as while reading the definition of the tax in the test's setUpClass method I can see the logic of it, instead of the implementation. 
Being explicit doesn't mean that I should repeat "factor_percent" every single line. And if I want to know what is that `100` about, I just read the function body.
But that's my opinion, you're the reviewer.

Reply:
👍🏻  as per Discord message